content/post/abortion-bill.en.md

```diff
@@ -0,0 +1,83 @@
+
+#### The extent of the tracking uncovered to some users
+
+This topic regarding messages content on social media is very interesting as U.S. law forces digital companies like Facebook to give access to their full users' data on request to governmental forces. And as we already know, social media knows their users better than their family or even themselves. So filtering or Big Data algorithms could be used by state police to flag and track people who underwent abortion outside of the state or worst, people who think (or are in the process of thinking) of doing so in the future. Worst science fiction scenarii in the manner of Inception movie are starting to become true. Regarding this, some naive people discover the extent of Google users’ spying when Google said recently that all their users' GPS waypoints that are related to an abortion center will be automatically erased from their servers[^8]. Meaning that Police could not claim unexisting (because erased) data. In addition, Google knows and memorizes a large history of geolocation data of their users, and not surprisingly, it also knows precisely to which building, company, trade or care center these coordinates correspond. To sum up, Google obviously knows from what address you start moving and what address is your destination, it knows what corresponds to these addresses (home, companies…). Maybe it was not a big deal when we thought that their business was to sell shoes and other goods, but their data reveals much more than your shopping. This is not new but this abortion scandal may have revealed it to more people.
```

Comment:
```suggestion
This topic regarding messages content on social media is very interesting as U.S. law forces digital companies like Facebook to give access to their full users' data on request to governmental forces. And as we already know, social media knows their users better than their family or even themselves. So filtering or Big Data algorithms could be used by state police to flag and track people who underwent abortion outside of the state or worse, people who think (or are in the process of thinking) of doing so in the future. Worst science fiction scenarii in the manner of Inception movie are starting to become true. Regarding this, some naive people discovered the extent of Google users’ spying when Google said recently that all their users' GPS waypoints that are related to an abortion center will be automatically erased from their servers[^8]. Meaning that Police could not claim unexisting (because erased) data. In addition, Google knows and memorizes a large history of geolocation data of their users, and not surprisingly, it also knows precisely to which building, company, trade or care center these coordinates correspond. To sum up, Google obviously knows from what address you start moving and what address is your destination, it knows what corresponds to these addresses (home, companies…). Maybe it was not a big deal when we thought that their business was to sell shoes and other goods, but their data reveals much more than your shopping. This is not new but this abortion scandal may have revealed it to more people.
```
Wouldn't "think(s) about " be better?
Plural on correspond?

Reply:
I changed the "of" to "about" but for the "correspond", "what" counts as singular so "corresponds"